lib/inputstreamhelper/__init__.py

```diff
@@ -301,7 +301,7 @@ def _update_widevine(self):
                 log(3, 'User declined to update {component}.', component=component)
         else:
             from time import mktime
-            set_setting('last_update', mktime(datetime.utcnow().timetuple()))
+            set_setting('last_update_check', mktime(datetime.utcnow().timetuple()))
```

Comment:
Can we not call this simply `last_check` ? It is less confusing IMO.

Reply:
Agree and done.